src/ducks/soknad.js

```diff
@@ -1,6 +1,7 @@
 import { createSelector } from 'reselect';
 import * as Api from '../services/api';
 import { STATUS, doThenDispatch } from '../services/utils';
+import { norskTilBool } from '../utils/utils';
```

Comment:
Verdiene i radioknapp er "ja" og "nei". Må diskutere med backend om vi får true || false, men for å unngå en haug løse tråder så foreslår jeg dette som løsning nå.